data/sp500.csv

```diff
@@ -0,0 +1,124 @@
+date,price
```

Comment:
What file is this???

Reply:
probably commited the csv for the previous slider 😅

Reply:
deleted all data files